tests/core/test_farmer_harvester_rpc.py

```diff
@@ -534,7 +540,8 @@ async def assert_added(path: Path) -> None:
         await harvester_rpc_client.fetch("add_plot_directory", {})
 
     root_path = harvester_service._node.root_path
-    test_path = Path(root_path / "test_path").resolve()
+    random_name = token_bytes(32).hex()
+    test_path = Path(root_path / f"test_path_{random_name}").resolve()
```

Comment:
`Path()`ing a `/` joined `Path` object is redundant.  Could just drop the `Path` though I would write it as:

```suggestion
    test_path = root_path.joinpath(f"test_path_{random_name}").resolve()
```

Though really if we want a temporary directory we shouldn't shove it into the root path at all.  `tmp_path` is a fixture for getting a temp dir for the test.  Then don't clean them up.  pytest manages that for you, albeit that it keeps a few runs around before deleting the old temporary directories.